utensor_cgen/backend/utensor/snippets/templates/snippets/rearch/op_missing.cpp

```diff
@@ -2,6 +2,7 @@
     FIXME: {{op_type}} currently not supported, you have to fill up this section or it won't compile
 
     1. Declare the operators of type {{op_type}}
+        - If you're using the ModelInterface, make sure you also initialize the operator in the constructor
```

Comment:
What do you mean by this? Are you trying to say, if someone adds additional constructors to not forget to initialize the operators?

Reply:
Ya, they may forget to declare the ops they add in the header, I assume.